app.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Please read the first instruction again. Can you make the PR with just the javascript code? Thanks.